GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -3152,6 +3153,23 @@ private void Message_Enter(object sender, EventArgs e)
                 StagedSelectionChanged(null, null);
             }
         }
+
+        internal TestAccessor GetTestAccessor()
+            => new TestAccessor(this);
+
+        public readonly struct TestAccessor
+        {
+            private readonly FormCommit _formCommit;
+
+            public TestAccessor(FormCommit formCommit)
+            {
+                _formCommit = formCommit;
+            }
+
+            public EditNetSpell Message => _formCommit.Message;
```

Comment:
:memo: There is likely no need to use `TestAccessor` for purposes beyond exposing controls. Anything else would be considered a red flag (likely misuse or unnecessarily complex).